src/core/document.js

```diff
@@ -1439,6 +1494,8 @@ class PDFDocument {
       IsXFAPresent: formInfo.hasXfa,
       IsCollectionPresent: !!catalog.collection,
       IsSignaturesPresent: formInfo.hasSignatures,
+      collectedSignatureCertificates:
```

Comment:
This field is only for (temporary?) telemetry, but doing it like this makes it visible in the API surface judging by the tests we have e.g. here: https://github.com/mozilla/pdf.js/blob/1ebc896b6f82373eb1b6d296de2d6154dc2993d7/test/unit/api_spec.js#L2216
We probably don't want that in order to be able to delete it again later on once the telemetry is no longer needed. Hence it seems better to limit the entire signature collection to the MOZCENTRAL build target so other builds are not affected by it and it's simpler to remove it later on.

If that's not feasible, we should probably at least indicate that it's private by prefixing it with an underscore (but preferably we don't expose this in the API surface at all).